cli/change-detection/index.md

```diff
@@ -21,6 +21,39 @@ E.g., to list all stacks that contain changes:
 terramate list --changed
 ```
 
+Since `v0.11.0` untracked and uncommitted files are also considered changes.
+
+::: info
+_Untracked file_ means Git doesn't know about its existence.
```

Comment:
```suggestion
An _Untracked file_ is a file that Git doesn't recongnize.
```

Reply:
I think "Git doesn't recognize" is not correct.
Other ways of saying:

- Untracked are files not added nor committed
- Untracked are files not tracked by Git index
- Untracked are files that have been created within your repo's working directory but have not yet been added to the repository's tracking index using the `git add` command.  (got this in the [Atlassian docs](https://www.atlassian.com/git/tutorials/undoing-changes/git-clean#:~:text=Untracked%20files%20are%20files%20that,using%20the%20git%20add%20command.)

what do you prefer?

Reply:
I think the second option is good: clear & concise.